athena-hbase/src/main/java/com/amazonaws/athena/connectors/hbase/HbaseMetadataHandler.java

```diff
@@ -278,4 +309,34 @@ protected Field convertField(String name, String glueType)
     {
         return GlueFieldLexer.lex(name, glueType);
     }
+
+    @Override
+    public GetTableResponse doGetQueryPassthroughSchema(BlockAllocator allocator, GetTableRequest request) throws Exception
+    {
+        queryPassthrough.verify(request.getQueryPassthroughArguments());
+        String schemaName = request.getQueryPassthroughArguments().get(HbaseQueryPassthrough.DATABASE);
+        String tableName = request.getQueryPassthroughArguments().get(HbaseQueryPassthrough.COLLECTION);
+
+        com.amazonaws.athena.connector.lambda.domain.TableName tableNameObj = new com.amazonaws.athena.connector.lambda.domain.TableName(schemaName, tableName);
```

Comment:
didn't we import this already? `com.amazonaws.athena.connector.lambda.domain.TableName;`

Reply:
Hi @chngpe We already have another import for TableName with different package (org.apache.hadoop.hbase.TableName;) so we can't use import.